src/commands/Social/daily.ts

```diff
@@ -48,8 +50,9 @@ export default class extends SkyraCommand {
 
 	private async claimDaily(message: KlasaMessage, nextTime: number) {
 		const money = this.calculateDailies(message);
-		const total = money + message.author.settings.get(UserSettings.Money);
-		await message.author.settings.update([[UserSettings.Money, total], [UserSettings.TimeDaily, nextTime]], {
+		const currentMoney = await message.author.settings.get(UserSettings.Money);
+		this.client.emit(Events.MoneyTransaction, message.author as KlasaUser, money, currentMoney, EconomyTransactionAction.Add, EconomyTransactionReason.Daily);
```

Comment:
You don't need to cast `message.author`, also remove the import above.
```suggestion
		this.client.emit(Events.MoneyTransaction, message.author, money, currentMoney, EconomyTransactionAction.Add, EconomyTransactionReason.Daily);
```